packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1780,12 +1780,17 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
         this.setState({ isOpen: false });
       }
 
+      let allowedKeys: number[] = [KeyCodes.tab, KeyCodes.escape];
+
+      if (this.props.allowArrowKeyInputWhenDisabled === true) {
```

Comment:
I think rather than adding another confusing prop, we should have a good behavior by default.

If the component takes no action as a result of an arrow key, it should NOT stop propagation.

So, if you're in a horizontal FocusZone, and arrow right over to a combobox, and it selects some text, and you arrow right again, selection will cancel and move your cursor to the end, I think. If your cursor is at the end and you have nothing selected, then no cancelation should occur, letting the keystroke bubble up to the zone to move to the next thing.

If you hit down, and the control is disabled, it should not be expanding the options list, and therefore has no action, so stopPropagation should not be called. This allows the focuszone to move along to the next item.